app/controllers/parent_interface/consent_forms/edit_controller.rb

```diff
@@ -16,13 +17,30 @@ def show
     end
 
     def update
+      model = @consent_form
```

Comment:
What's the point of this assignment? I looks like it always gets overwritten on line 29.

Reply:
Sorry, nvm, I didn't think this method would be so long, didn't realise the `render model` at the end was part of this same method!